src/test/scala/com/microsoft/hyperspace/util/ResolverUtilsTest.scala

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.util
+
+import org.apache.spark.SparkFunSuite
+
+import com.microsoft.hyperspace.{HyperspaceException, SparkInvolvedSuite}
+
+class ResolverUtilsTest extends SparkFunSuite with SparkInvolvedSuite {
+
+  test("testResolve against dataframe - simple") {
```

Comment:
nit: ditto for all test case names, so that they look descriptive.

```suggestion
  test("Verify testResolve against dataframe - simple.") {
```

Reply:
I will do this change, although adding `"Verify"` word is redundant. The goal of an UT is to **verify** the function's behaviour.